plots.cabal

```diff
@@ -87,3 +87,26 @@ library
   ghc-options:         -Wall
 
   default-language:  Haskell2010
+
+-- are plot example
```

Comment:
yep, sorry! I was just trying to get a quick-and-dirty PR against the branch that I can iterate on